src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/Interfaces/IRemoveOrganizationUserCommand.cs

```diff
@@ -1,14 +1,53 @@
-﻿using Bit.Core.Entities;
-using Bit.Core.Enums;
+﻿using Bit.Core.Enums;
 
 namespace Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.Interfaces;
 
 public interface IRemoveOrganizationUserCommand
 {
+    /// <summary>
+    /// Removes a user from an organization.
+    /// </summary>
+    /// <param name="organizationId">The ID of the organization.</param>
+    /// <param name="userId">The ID of the user to remove.</param>
+    Task RemoveUserAsync(Guid organizationId, Guid userId);
```

Comment:
Its not really a bug though -- its different from the other methods because it retrieves the `OrganizationUser` object by filtering by both `OrganizationId` **and** `UserId`.

I can create the follow-up ticket. I didn't want to refactor too much here, my intention was just to tidy this up a bit whilst adding the new error if the user is managed.

Reply:
My bad! This showed up on the diff as a new line so I thought you were adding this interface, which would've meant that existing callers in `main` today were calling one of the other methods with a `UserId`. But you just moved it. Disregard what I said about there being a bug - but please still create a follow up ticket :)

Reply:
Ticket created [here](https://bitwarden.atlassian.net/browse/PM-15427)